src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -894,12 +894,14 @@ bool main_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
         add_slave_socket(sd, strIfaceName);
 
         work_queue.enqueue<void>([this, strIfaceName, sd]() {
-            // Response message (empty for now)
-            size_t buffer_size = message_com::get_vs_cmdu_size_on_buffer<
-                beerocks_message::cACTION_PLATFORM_SON_SLAVE_REGISTER_RESPONSE>();
+            size_t headroom = sizeof(beerocks::message::sUdsHeader);
+            size_t buffer_size =
+                headroom + message_com::get_vs_cmdu_size_on_buffer<
+                               beerocks_message::cACTION_PLATFORM_SON_SLAVE_REGISTER_RESPONSE>();
             uint8_t tx_buffer[buffer_size];
-            ieee1905_1::CmduMessageTx cmdu_tx(tx_buffer, sizeof(tx_buffer));
+            ieee1905_1::CmduMessageTx cmdu_tx(tx_buffer + headroom, buffer_size - headroom);
```

Comment:
I generally prefer to use `sizeof(tx_buffer)` instead of `buffer_size` (like it was originally). That way, if you change the size of the buffer, it remains correct.

Reply:
Done